book/src/drivers.md

```diff
@@ -88,6 +88,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [SPDK-CSI](https://github.com/spdk/spdk-csi) | `csi.spdk.io` | v1.1 | A Container Storage Interface (CSI) Driver for [SPDK](https://spdk.io/) | Persistent and Ephemeral | Read/Write Single Pod | Yes |
 [StorageOS](https://docs.storageos.com/docs/platforms/kubernetes/install/) | `storageos` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [StorageOS](https://storageos.com/) | Persistent | Read/Write Multiple Pods | Yes |
 [Storidge](https://docs.storidge.com/kubernetes_storage/overview.html) | `csi.cio.storidge.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for [Storidge CIO](https://storidge.com/) | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion
+[StorPool](https://storpool.com/kubernetes) | `csi-driver.storpool.com` | v1.0 | A Container Storage Interface (CSI) Driver for [StorPool](https://storpool.com/) | Persistent and Ephemeral | Read/Write Multiple Pods | Yes | Expansion
```

Comment:
Looks like "https://storpool.com/kubernetes" only has a very high level overview.  Do you have a website that shows how to use your CSI driver?

Reply:
Yes, it is here - https://kb.storpool.com/storpool_integrations/github/kubernetes.html

Reply:
Great.  Can you use this link instead of "https://storpool.com/kubernetes"?

Reply:
Done.